azure-pipelines.yml

```diff
@@ -60,6 +60,20 @@ stages:
           Write-Host "##vso[task.setvariable variable=SONAR_PROJECT_VERSION;]$sonarProjectVersion"
         displayName: 'Read Sonar project version from the versions.props file'
 
+      - powershell: |
+          cd sonaranalyzer-dotnet
+          $testProjFileName = "tests\SonarAnalyzer.UnitTest\SonarAnalyzer.UnitTest.csproj"
+
+          Write-Host "Remove .Net 5 target for UnitTest project"
```

Comment:
So we will later configure the environment. I think this is fine since it will be much easier after the official release. 

Is there an issue to not forget about this?

Reply:
I opened https://github.com/SonarSource/sonar-dotnet/issues/3684